CMakeLists.txt

```diff
@@ -71,7 +71,7 @@ project(Daemon C CXX)
 
 if(NOT DAEMON_EXTERNAL_APP)
     option(BUILD_CLIENT "Engine client executable, required to play" ON)
-    option(BUILD_SERVER "Engine server executable, required to host servers" ON)
+    option(BUILD_SERVER "Engine server executable, required to host servers" OFF)
```

Comment:
I don't think server or or ttyclient should be turned off.

The nacl and native exe changes are good.

Reply:
what are server and tty client useful for when you don't build the shared modules anyway?

Reply:
Hosting a server with DPKs from the release.